lib/net/http/header.rb

```diff
@@ -923,12 +923,22 @@ def set_form(params, enctype='application/x-www-form-urlencoded', formopt={})
     end
   end
 
-  # Set the Authorization: header for "Basic" authorization.
+  # Sets header <tt>'Authorization'</tt> using the given
+  # +account+ and +password+ strings:
+  #
+  #   req.basic_auth('my_account', 'my_password')
+  #   req['Authorization'] # => ["Basic bXlfYWNjb3VudDpteV9wYXNzd29yZA=="]
```

Comment:
This returns a string, not an array.

```suggestion
  #   req['Authorization'] # => "Basic bXlfYWNjb3VudDpteV9wYXNzd29yZA=="
```

Reply:
Fixed.  (This was slovenly work on my part.)